### PR TITLE
SHS 137: Investigate Views/module issue where alt text is omitted 

### PR DESCRIPTION
**Pull request by hectorlj**
# READY FOR REVIEW

## Summary
Image alt text now displays on images that come from views.

## Urgency
_low_

## Steps to Test
1. Go to /news and /events
2. Pictures with existing alt text should have it populate the `alt` attribute.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

## Notes
_Tested locally by checking last picture on [https://west.suhumsci.loc/news/center-news-and-notes](https://west.suhumsci.loc/news/center-news-and-notes)_

### CODE REVIEW

File: docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile [hunk 1]
```diff
@@ -173,7 +173,6 @@ function su_humsci_profile_preprocess_image_formatter(&$variables) {
   if (isset($variables['url'])) {
     // Disable screen readers from seeing the link on the image since there
     // should be another link with text nearby.
-    $variables['image']['#alt'] = '';
```
Comment (callinmullaney) [inline docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile:176]: @hectorlj Good find! However, the comments on this code make it seem intentional. I think we may also need to add `aria-hidden="true"` so screen readers truely skip over the image.

**Review by callinmullaney — CHANGES_REQUESTED** [review]
See code comment. Nice work tracking this one down. I think the original approach here was flawed in trying to get screen reader to skip linked images.

File: docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile (hunk 1 above)
Comment (hectorlj) [reply docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile:176]: Fixed for both preprocessor functions.

**Review by callinmullaney — DISMISSED** [review]
Thanks @hectorlj These changes look good to me and are correctly excluding these images from screen-readers.

**Comment by callinmullaney** [comment]
@pookmish This PR is ready to merge into the 9.4.6 release. Small change that properly omitted linked images from screen readers VS just detaching the alt attribute. Thanks!

**Pull request merged**